src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/helpers/SpringAPIServiceTest.kt

```diff
@@ -43,6 +43,6 @@ import kotlin.annotation.AnnotationTarget.CLASS
   SentencingAdjustmentsApiExtension::class,
 )
 @ActiveProfiles("test")
-@SpringBootTest(classes = [WebClientConfiguration::class, WebClientAutoConfiguration::class, OAuth2ClientAutoConfiguration::class, SecurityAutoConfiguration::class])
+@SpringBootTest(classes = [WebClientConfiguration::class, WebClientAutoConfiguration::class, ReactiveOAuth2ClientAutoConfiguration::class, ReactiveSecurityAutoConfiguration::class])
```

Comment:
this caused me no end of grief.  Spent ages working out why the test wouldn't find the reactive auto configuration, hadn't realised that it had been hard coded here 😠